sig-network/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The primary objective of the SIG is to connect the Virtual Machine to the pod network/s,
```

Comment:
there is only one pod network, not?

Would it be fair to also include 2ndary networks in this charter?

Reply:
The pod may have primary and secondary networks. Changed to - `The main objective of the SIG is to connect the Virtual Machine to the primary and secondary pod networks,`.
I hope it is clearer.